frontend-src/Miso/Html/Internal.hs

```diff
@@ -130,7 +130,10 @@ node ns tag key attrs kids = View $ \sink -> do
   set "tag" tag vnode
   set "key" key vnode
   setAttrs vnode sink
-  flip (set "children") vnode =<< ghcjsPure . jsval =<< setKids sink
+  flip (set "children") vnode
+    =<< collapseSiblingTextNodes
```

Comment:
It looks like the children are being collapsed in ghcjs before every diff. Is this necessary? Can we get away with doing it only on the initial sync up between the server sent html and first render?

I'm not sure there were diffing problems between double text nodes when not using isomorphism, so feel free to shoot this idea down immediately 😅

Reply:
This is a good idea, we shouldn't call collapse on each diff iteration.